lighthouse-core/audits/metrics.js

```diff
@@ -115,7 +120,7 @@ class Metrics extends Audit {
     const details = {
       type: 'debugdata',
       // TODO: Consider not nesting metrics under `items`.
-      items: [metrics],
+      items: [metrics, {invalidLcp: traceOfTab.invalidLcp}],
```

Comment:
putting a boolean property on `metrics` really screws with TS.

Reply:
> putting a boolean property on `metrics` really screws with TS.

good, let's keep it out of there :P